float64[] elements
string node_name
string consumer_name
float64 timestamp